Providers/Scripts/2.4x-2.5x/Scripts/nxOMSKeyMgmt.py

```diff
@@ -0,0 +1,334 @@
+#!/usr/bin/env python
+#============================================================================
+# Copyright (C) Microsoft Corporation, All rights reserved.
+#============================================================================
+
+import os
+import sys
+import subprocess
+import imp
+import re
+import base64
+
+# class MSFT_nxOMSKeyMgmt:OMI_BaseResource
+# {
+#   [Key] string KeyContents;
+#   [Write,required] string KeySignature;
+#   [Write,ValueMap{"present", "absent"},Values{"present", "absent"}] string Ensure;
+# };
+
+protocol = imp.load_source('protocol', '../protocol.py')
+nxDSCLog = imp.load_source('nxDSCLog', '../nxDSCLog.py')
+
+LG = nxDSCLog.DSCLog
+
+dsc_keyring_path = '/etc/opt/omi/conf/omsconfig/keyring.gpg'
+signature_keyring_path = '/etc/opt/omi/conf/omsconfig/keymgmtring.gpg'
+gpg_bin = '/usr/bin/gpg'
```

Comment:
Good point!.  I'm sending the '--options /dev/null' which skips the sourcing of the gpg options file for the user,
so I think it's OK to skip exporting $HOME.